lib/src/main/java/io/cloudquery/scalar/JSON.java

```diff
@@ -0,0 +1,82 @@
+package io.cloudquery.scalar;
+
+import static io.cloudquery.scalar.ValidationException.NO_CONVERSION_AVAILABLE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Should `jackson` be added as a dependency to `build.gradle`?

Unrelated, but I used `gson` in https://github.com/cloudquery/plugin-sdk-java/pull/81/files#diff-655a69127303f6948c0b150902436756156ec7f82640e994c1f552cbdec5bbceR48

Reply:
Do you have a preference. I think I saw some hints that Jackson is slightly faster, which is why I went with it, but I have no solid preference.